packages/core/useFavicon/index.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import 'vue-tsx-support/enable-check'
+import Vue from 'vue'
+import { storiesOf } from '@storybook/vue'
+import { defineComponent } from 'vue-demi'
+import { ShowDocs } from '../../_docs/showdocs'
+
+const Demo = defineComponent({
```

Comment:
You can use the new style to define the stories. Refer to `packages/core/_template/index.stories.tsx`